.mailmap

```diff
@@ -1582,6 +1582,7 @@ anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.n
 anutosh491 <andersonbhat491@gmail.com> anutosh491 <87052487+anutosh491@users.noreply.github.com>
 arooshiverma <av22@iitbbs.ac.in>
 atharvParlikar <atharvparlikar@gmail.com>
+bharatAmeria <21001019007@jcboseust.ac.in> bharatAmeria <147488804+bharatAmeria@users.noreply.github.com>
```

Comment:
This should be your real name like:
```
Bharat Ameria <email address> bharatAmeria <21001019007@jcboseust.ac.in> bharatAmeria <147488804+bharatAmeria@users.noreply.github.com>
```
@smichr before merging please ask someone whether they actually want this name to be the one that gets recorded in the AUTHORS file.

@bharatAmeria is this the name that you would to have recorded in the AUTHORS file?

Reply:
Bharat Ameria this is my real name. Should, I change this name